EDMarketConnector.py

```diff
@@ -510,7 +510,7 @@ def retry_for_shipyard(self):
         except:
             pass
 
-    def system_change(self, timestamp, system):
+    def system_change(self, timestamp, system, coordinates):
```

Comment:
I had to change line 521 `plug.notify_system_changed(timestamp, system)` to `plug.notify_system_changed(timestamp, system, coordinates)` otherwise it caused an exception when I jumped to a new system.